cli/popper/commands/cmd_search.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+
+import click
+
+from popper import utils as pu
+from popper.cli import log, pass_context
+
+
+@click.argument(
+    'keywords',
+    required=True
+)
+@click.option(
+    '--update-cache',
+    help=('Skip updating the metadata and search on the local cache.'),
```

Comment:
The message should be something like `Update the metadata cache before searching`